manim/mobject/svg/text_mobject.py

```diff
@@ -77,6 +77,14 @@ class Text(SVGMobject):
         `weird <https://github.com/3b1b/manim/issues/1067>`_. Consider using
         :meth:`remove_invisible_chars` to resolve this issue.
 
+    TESTS
+    -----
+
+    Check whether writing text works::
+
+        >>> Text("The horse does not eat cucumber salad.")
+        <manim.mobject.svg.text_mobject.Text object at ...>
```

Comment:
Personally, I don't like testing for the default string representation of Python classes that much and I would prefer to introduce a lightweight `__repr__` for these kind of things. My suggestion would be something like

```py

def __repr__(self):
    return f'Text("{self.original_text}")'
```
(or maybe also without the quotes or parentheses... I'm open for suggestions) so that we would have
```suggestion
        Text("The horse does not eat cucumber salad.")
```
as output for the doctest.

Any opinions on `__repr__`?

Reply:
The changed one seems better to me